packages/ceramic-doctype-tile/src/__tests__/three-id.test.ts

```diff
@@ -49,6 +49,7 @@ const RECORDS = {
         controllers: [
           "did:key:zQ3shwsCgFanBax6UiaLu1oGvM7vhuqoW88VBUiUTCeHbTeTV"
         ],
+        isUnique: false,
```

Comment:
Can you edit the Tile doctype handler to make sure that if `isUnique` is set to `false` then the property is **not** included in the header of the genesis record?

Reply:
If we do that, then documents can no longer roundtrip correctly.  I.E. if you copy an existing document and then create a new document with the copied output as the arguments, now you get different behavior (a unique document where the document you copied was global).  Given that users can provide their own metadata objects directly, modifying the input they provide out from under them seems fishy.

I actually kind of feel like either we should *always* include isUnique in the metadata, or *never* include it.  Including it conditionally based on its value seems brittle and conceptually unclear.  Either the uniqueness of a document is something that is inherently important to know about any given document, or it isn't.  I wonder if rather than considering 'isUnique' to be part of a document's metadata, instead it should just be considered an argument to the various functions for creating documents.  We already have the separate 'unique' field which ensures the behavior is correct, conceptually I'm not sure uniqueness is part of an object's inherent description/metadata and think it may just be some syntactic sugar to the document creation process.

Reply:
Good point. Maybe we can set `unique` to 0 if false? It's a little strange to have two different representations here. Would be nice if it's always one field.